README.md

```diff
@@ -1,13 +1,15 @@
 # Jenkins MATLAB Plugin
 
-The Jenkins plugin for MATLAB&reg; enables you to easily run your MATLAB and Simulink&reg; tests and generate artifacts such as *JUnit* and *TAP* test results, and *Cobertura* code or model coverage reports. You can also export your Simulink Test&#8482; Manager results. This guide shows you examples of how to configure the plugin for different objectives, considering a Microsoft&reg; Windows&reg; platform.
+The Jenkins plugin for MATLAB&reg; enables you to easily run your MATLAB and Simulink&reg; tests and generate artifacts such as *PDF* test report, *JUnit* and *TAP* test results, and *Cobertura* code or model coverage reports. You can also export your Simulink Test&#8482; Manager results. This guide shows you examples of how to configure the plugin for different objectives, considering a Microsoft&reg; Windows&reg; platform.
+
+*Note: Producing a PDF test report is not supported on Mac OS platform.*
```

Comment:
Do we really need to the put the note here or can we just warn in the GUI?

In any case, the sentence can be tweaked as follows:

Note: Producing a PDF test report is currently not supported on MacOS platforms.

Reply:
Please use bold font for Note: 

**Note:**

Reply:
Yes, I agree that we don't need to put the note here. It might create unwanted attention.